libclambcc/Common/CMakeLists.txt

```diff
@@ -0,0 +1,74 @@
+# Copyright (C) 2021 Cisco Systems, Inc. and/or its affiliates. All rights reserved.
+
+#
+# The clambccommon object library
+#
+add_library(clambccommon_obj OBJECT)
+target_sources(clambccommon_obj
+    PRIVATE
+        ClamBCDiagnostics.cpp
+        ClamBCUtilities.cpp
+        version.c
```

Comment:
Looks like ClambCRegAlloc.cpp was copied out and is not in this list, but it still exists in this directory.  Should it be deleted from `Common/`?

Reply:
done